### PR TITLE
Whats new in Operator Connect

### PR DESCRIPTION
**Pull request by msreddy05**

### CODE REVIEW

File: Teams/operator-connect-whats-new.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+# What's new for Operator Connect
+
+This article describes what's new in Operator Connect. Check back often for updates.
+
+## DR to OC Migration API
+
+On July 1, 2022, Microsoft published an API for operators where they can migrate the numbers from Direct Routing to Operator Connect. 
+
+For more information, see Operator Connect Swagger https://operatorconnect.microsoft.com/api/swagger#tag/TnMigration-(New).
```
Comment (JohanFreelancer9) [inline Teams/operator-connect-whats-new.md:28]:
```suggestion
For more information, see [Operator Connect Swagger](https://operatorconnect.microsoft.com/api/swagger#tag/TnMigration-(New)).
```

File: Teams/operator-connect-whats-new.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Number Status
+
+On July 1, 2022, Microsoft published an API for operators which is used to provide the operator status information about the given numbers with respect to the consented tenant.
+
+For more information, see Operator Connect Swagger https://operatorconnect.microsoft.com/api/swagger#operation/NumberStatus_GetNumberStatus.
```
Comment (JohanFreelancer9) [inline Teams/operator-connect-whats-new.md:34]:
```suggestion
For more information, see [Operator Connect Swagger](https://operatorconnect.microsoft.com/api/swagger#operation/NumberStatus_GetNumberStatus).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msreddy05, kindly commit the suggested changes. Many thanks.

**Review by kushkgp — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@msreddy05 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by msreddy05** [comment]
@tiburd Can you please merge the changes

**Pull request merged**